src/utilities/urls.ts

```diff
@@ -0,0 +1,13 @@
+export const normalizeUrl = (string: string): string => {
+  if (!string) {
+    return ''
+  }
```

Comment:
I don't think this util should be responsible for validating that what it's being given is a url. That said, if an empty string is given to it then it would return `http://` without this check, which felt weird. This much of a guard made sense to me but happy to change this up if needed.

Reply:
Makes sense to me 👍